### ISSUE TITLE
configure: error: WRONG!  uint32_t not defined correctly. 

### ISSUE THREAD

**Issue by kkoomen** [rhysd/action-setup-vim#27]
Hi,

See my github action: https://github.com/kkoomen/vim-doge/actions/runs/6519489351/job/17706041103

```
 checking size of long... 8
checking size of time_t... 8
checking size of off_t... 8
checking uint32_t is 32 bits... configure: error: WRONG!  uint32_t not defined correctly.
Error: Command './configure --prefix=/Users/runner/vim-v8.0.0000 --with-features=huge --enable-fail-if-missing' exited non-zero status 1: configure: error: WRONG!  uint32_t not defined correctly. 
Error: Command './configure --prefix=/Users/runner/vim-v8.0.0000 --with-features=huge --enable-fail-if-missing' exited non-zero status 1: configure: error: WRONG!  uint32_t not defined correctly.
```

This is also occurring for vim v7.4 builds, can you fix this?

**Comment by rhysd** [comment]
That's known issue of combination of Xcode and old Vim. So it is not an issue of this action.

- https://github.com/vim/vim/pull/6381
- https://github.com/vim/vim/commit/5289783e0b07cfc3f92ee933261ca4c4acdca007

To build Vim older than 8.2.1119, Xcode 11 or earlier is necessary.

**Comment by rhysd** [comment]
I confirmed `/Applications/Xcode_11.7.app` exists on `macos-11` runner. Please use it. When the `Xcode_11.7.app` exists, action-setup-vim automatically tries to use it for building Vim older than 8.2.1119.

https://github.com/rhysd/action-setup-vim/blob/f980ef4c7e89828798940f25f5c58f653bb579e1/src/vim.ts#L70-L85

**Comment by kkoomen** [comment]
Hi,

Thanks for responding quickly. I still have issues with macos-11 for v7.4.2119 and v8.0.0000:

- v7.4.2119: https://github.com/kkoomen/vim-doge/actions/runs/6522599262/job/17712340465
- v8.0.0000 https://github.com/kkoomen/vim-doge/actions/runs/6522649211/job/17712442668

```
1422 translated messages.
1735 translated messages.
1883 translated messages.
make[1]: *** [languages] Error 2
make[1]: *** Waiting for unfinished jobs....
make: *** [first] Error 2
Error: Command 'make -j' exited non-zero status 2: 1898 translated messages. 1284 translated messages. 1728 translated messages. 1628 translated messages. 1883 translated messages. 78 translated messages. 1898 translated messages. zh_CN.cp936.po:2899: zh_CN.cp936.po: input is not valid in "GBK" encoding make[2]: *** [zh_CN.cp936.mo] Error 1 make[2]: *** Waiting for unfinished jobs.... 1768 translated messages. 1213 translated messages. 1895 translated messages. 1869 translated messages. 1628 translated messages. 1897 translated messages. 1668 translated messages. 1895 translated messages. 1423 translated messages. 1422 translated messages. 1882 translated messages. 1701 translated messages. 1668 translated messages. 1888 translated messages. 1634 translated messages, 3 fuzzy translations. 1882 translated messages. 1422 translated messages. 1733 translated messages. 1633 translated messages, 3 fuzzy translations. 1728 translated messages. 1697 translated messages. 1898 translated messages. 1697 translated messages. 1869 translated messages. 1869 translated messages. 1284 translated messages. 1422 translated messages. 1735 translated messages. 1883 translated messages. make[1]: *** [languages] Error 2 make[1]: *** Waiting for unfinished jobs.... make: *** [first] Error 2 
Error: Command 'make -j' exited non-zero status 2: 1898 translated messages. 1284 translated messages. 1728 translated messages. 1628 translated messages. 1883 translated messages. 78 translated messages. 1898 translated messages. zh_CN.cp936.po:2899: zh_CN.cp936.po: input is not valid in "GBK" encoding make[2]: *** [zh_CN.cp936.mo] Error 1 make[2]: *** Waiting for unfinished jobs.... 1768 translated messages. 1213 translated messages. 1895 translated messages. 1869 translated messages. 1628 translated messages. 1897 translated messages. 1668 translated messages. 1895 translated messages. 1423 translated messages. 1422 translated messages. 1882 translated messages. 1701 translated messages. 1668 translated messages. 1888 translated messages. 1634 translated messages, 3 fuzzy translations. 1882 translated messages. 1422 translated messages. 1733 translated messages. 1633 translated messages, 3 fuzzy translations. 1728 translated messages. 1697 translated messages. 1898 translated messages. 1697 translated messages. 1869 translated messages. 1869 translated messages. 1284 translated messages. 1422 translated messages. 1735 translated messages. 1883 translated messages. make[1]: *** [languages] Error 2 make[1]: *** Waiting for unfinished jobs.... make: *** [first] Error 2 
```

Is this fixable?

**Comment by rhysd** [comment]
That's a build error of Vim. It is not related to this action.

**Comment by rhysd** [comment]
I tried some `./configure` options. It seems that adding `--disable-nls` can avoid the `.po` files generation. I'll try to add new input for this.

**Comment by rhysd** [comment]
@kkoomen I released [v1.3.0](https://github.com/rhysd/action-setup-vim/releases/tag/v1.3.0) with new `configure-args` input. Now you can disable NLS and it can avoid the build issue.

```yaml
- uses: rhysd/action-setup-vim@v1
  with:
    version: 8.0.0000
    configure-args: |
      --with-features=huge --enable-fail-if-missing --disable-nls
```

I confirmed Vim 8.0.0000 was successfully built with the above inputs on CI: https://github.com/rhysd/action-setup-vim/actions/runs/6523761712/job/17714720585#step:6:71

**Comment by kkoomen** [comment]
Works! https://github.com/kkoomen/vim-doge/actions/runs/6523862763/job/17714925034

Thanks a lot for your fast replies and quick fixes, really appreciate it!
